### ISSUE TITLE
Code modification problem

### ISSUE THREAD

**Issue by ElliottZhen** [LPXTT/SimTrack#13]
Hello author, may I ask if you ever encountered the error of DDP backpropagation when modifying the code on the basis of stark, such as the picture? I also modified on the basis of stark following your code style, and the error was located at loss.backward(), which could not be done with a single card? I'm very sorry to bother you. Looking forward to your reply.
![(ANKY2LS2 NZH 3C%E` KLD_tmb](https://github.com/LPXTT/SimTrack/assets/56305119/868f14d5-be2b-4177-9dd9-d540cad50c79)

**Comment by LPXTT** [comment]
Hello Wanzhen, I apologize for the inconvenience, but I haven't come across this issue previously, and as a result, I'm unable to provide any assistance. I'm sorry for any frustration this may cause.

**Comment by ElliottZhen** [comment]
Hello, I'm sorry for forgetting to reply you. Finally, I carefully read your code and solved the problem. I forgot to set the broadcast parameters of DDP.



---Original---
From: ***@***.***&gt;
Date: Thu, Jul 13, 2023 10:28 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [LPXTT/SimTrack] Code modification problem (Issue #13)




 
Hello Wanzhen, I apologize for the inconvenience, but I haven't come across this issue previously, and as a result, I'm unable to provide any assistance. I'm sorry for any frustration this may cause.

**Comment by ElliottZhen** [comment]
Thank you for your open source, so that people with poor code ability like me can have reference resources, I wish you a smooth scientific research!



---Original---
From: ***@***.***&gt;
Date: Thu, Jul 13, 2023 10:28 AM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [LPXTT/SimTrack] Code modification problem (Issue #13)




 
Hello Wanzhen, I apologize for the inconvenience, but I haven't come across this issue previously, and as a result, I'm unable to provide any assistance. I'm sorry for any frustration this may cause.
